plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
+
+
+def generate_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--selector', type=str, action="store",
+                        required=True, help="Label Selector")
+    return parser
+
+
+check_mongodb_cmd = ("bash", "-c",
+                     "mongo 127.0.0.1/admin "
+                     "-u \"admin\" -p \"$MONGODB_ADMIN_PASSWORD\" "
+                     "--eval=\"rs.status().myState\" --quiet")
```

Comment:
Yeah, I had that at one point, but I wasn't sure about mixing double quoting and single quoting for different parameters on the same call, but I do like not having to do the escaping.  As for the length, I only broke it up because of flake8's complaint.  I'll go with this and just ignore any complaint from flake8.